.authors.yml

```diff
@@ -8,7 +8,7 @@
   aliases:
   - Mike Sarahan
   - Michael Sarahan
-  num_commits: 2002
+  num_commits: 4004
```

Comment:
This means you're running the last release of `rever` and not the development branch which corrects the duplicate counting (see https://github.com/regro/rever/pull/266). To correct this please install the development branch of `rever` and rerun:

```bash
$ conda create -n rever --only-deps rever
$ conda activate rever
(rever) $ git clone https://github.com/regro/rever.git
(rever) $ cd rever
(rever) $ pip install .
```